src/agri_ai/core/master_agent.py

```
@@ -0,0 +1,373 @@
+
+        # データベース接続
+        # MongoDB接続は非同期処理で実行
+        import asyncio
+
+        if not mongodb_client.is_connected:
+            try:
+                # 既存のイベントループを確認
+                try:
+                    loop = asyncio.get_running_loop()
+                    # 既にイベントループが実行中の場合はタスクとしてスケジュール
+                    task = loop.create_task(mongodb_client.connect())
```

Ruff (0.12.2)

Local variable `task` is assigned to but never used

Remove assignment to unused variable `task`

(F841)